cmd/kubelet/app/server.go

```diff
@@ -379,11 +380,12 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 			}
 		}
 
+		nodeName, err = getNodeName(cloud, nodeutil.GetHostname(s.HostnameOverride))
+		if err != nil {
+			return err
+		}
+
 		if s.BootstrapKubeconfig != "" {
-			nodeName, err := getNodeName(cloud, nodeutil.GetHostname(s.HostnameOverride))
```

Comment:
Oh,I just think we can get nodeName first and then use nodeName somewhere, here use nodeName in s.BootstrapKubeconfig != "" check and maybe somewhere else.
I can move it back if my understanding is wrong.